docs/examples/memoryviews/memview_to_c.pyx

```diff
@@ -0,0 +1,27 @@
+# distutils: sources=./C_func_file.c
+# distutils: include_dirs=./
+
+cdef extern from "C_func_file.h":
+    void multiply_by_10_in_C(double *, unsigned int)
+
+import numpy as np
+
+def multiply_by_10(arr): # 'arr' is a one-dimensional numpy array
+
+    if not arr.flags['C_CONTIGUOUS']:
+        arr = np.ascontiguousarray(arr) # Makes a contiguous copy of the numpy array.
+
+    cdef double[::1] arr_memview = arr

```

Comment:
Would be nice to have a short comment here that `::1` requests a C contiguous view, and fails if the buffer is not contiguous. For those who didn't come across this notation yet.

Reply:
To avoid cluttering the Cython example, I've put it below.